### PR TITLE
Task75 imu compensation package 209 217

### PR DESCRIPTION
**Pull request by 7ofNine**
How about task #67 Transcribe LIST PROCESSING INTERPRETER as next one.
I know it is a big lump.

### CODE REVIEW

**Comment by jimlawton** [comment]
@7ofNine if you're doing `LIST PROCESSING INTERPRETER`, you could use the later interpreter in LUMINARY099 as a starting point, i.e. https://github.com/rburkey2005/virtualagc/blob/master/Luminary099/INTERPRETER.agc.

**Comment by 7ofNine** [comment]
The branch is back were it is supposed to be. I accidentally pushed new parts into the wrong branch.
The revert #1f84237 fixes that

File: Aurora12/IMU_COMPENSATION_PACKAGE.agc [hunk 1]
```diff
@@ -25,3 +26,419 @@
 #       https://www.ibiblio.org/apollo.  
 # The original high-quality digital images are available at archive.org:
 #       https://archive.org/details/aurora00dapg
+
+##Page 209
+                BANK    7
+                EBANK=  NBDX
+# PROGRAM DESCRIPTION- IMU COMPENSATION (LEM)                             DATE- 30 AUG 66
+# MOD NO- 0                                                               LOG SECTION- IMU COMPENSATION PACKAGE
+# MOD BY- GILBERT                                                         ASSEMBLY- SUNBURST REVISION 13
```
Comment (thewonderidiot) [inline Aurora12/IMU_COMPENSATION_PACKAGE.agc]: Code from Sunburst 13 made it back into here? That's really interesting!

**Comment by thewonderidiot** [comment]
Awesome, thanks! This closes out #75.

**Pull request merged**